src/library_glfw.js

```diff
@@ -1707,7 +1741,21 @@ var LibraryGLFW = {
 
   glfwGetClipboardString: function(win) {},
 
-  glfwMakeContextCurrent: function(winid) {},
+  glfwMakeContextCurrent__deps: ['emscripten_webgl_make_context_current'],
```

Comment:
Why is this dependency needed here?

Reply:
Is it enough to define it for GLFW?

Reply:
nvm. i see now.

Reply:
Fixed: adb002bd556089029753a099962ebea29cb349f5